components/core/tools/scripts/lib_install/spdlog.sh

```diff
@@ -29,8 +29,16 @@ fi
 
 # Check if already installed
 set +e
-dpkg -l ${package_name} | grep ${version}
-installed=$?
+pkg-config --modversion "spdlog = 1.9.2" >/dev/null 2>&1
+pkg_found=$?
+if [ $pkg_found -eq 0 ] ; then
+  find /usr/lib/ /usr/local/lib/ -name 'libspdlog.a' | grep . >/dev/null 2>&1
+  static_lib_found=$?
+fi
+if [ $static_lib_found -eq 0 ] ; then
+  dpkg -l ${package_name} | grep ${version}
```

Comment:
Is this still necessary?

Reply:
We probably want to keep it, unless we don't want the `libspdlog-dev` dpkg info to be displayed as done by the other scripts. 

On the other hand, it's true the above `pkg-config` and `find` checks should be able to guarantee the new CMake spdlog find script to find the right static library. The method is also platform (e.g. Debian / Ubuntu) agnostic as well.